Sources/HeliumLogger/HeliumLogHandler.swift

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright IBM Corporation 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Logging
+
+public extension HeliumLogger {
```

Comment:
Although this is correct, my preference would be to remove the `public` default access level from the extension and apply it to each of the functions that are public API.  (It makes it easier to tell whether future modifications affect the public API or not).